### DISCUSSION TITLE
Accessing AndroidX.AppCompat.Widget.Toolbar in NavigationPage

### DISCUSSION THREAD

**Discussion by AlleSchonWeg** [dotnet/maui#21637] [General] [answered]
Hi,
i want to draw a separator line between my content and my TitleView on Android. In Xamarin i created a custom NavigationPageRenderer, override OnViewAdded and if the view is AndroidX.AppCompat.Widget.Toolbar  i add a Drawable to the ToolBar Foreground.
How can i do this with MAUI? I tried `Microsoft.Maui.Handlers.NavigationViewHandler.Mapper.AppendToMapping(...)` but i can't find the native Toolbar.

**Comment by Pastajello** [comment, marked as answer]
You can search for the appbar on the activity.
FindViewById<LinearLayout>(Resource.Id.navigationlayout_appbar);. 
First child of it should be a MaterialToolbar.

**Reply by AlleSchonWeg** [reply]
Thank you. It works:
`Platform.CurrentActivity.FindViewById(Resource.Id.navigationlayout_appbar)`
